README.md

```diff
@@ -1,6 +1,6 @@
 <img alt="MIMIC" src="https://i.imgur.com/eodn4M4.png" height="75">
 
-Mimic is an API-compatible mock service for  __Openstack Compute__ and __Rackspace's implementation of Identity and Cloud Load balancers__. It is backed by in-memory data structure rather than a potentially expensive database.
+Mimic is an API-compatible mock service for  __Openstack Compute__, __Rackspace's implementation of Identity and Cloud Load balancers__, and __more__. It is backed by in-memory data structure rather than a potentially expensive database.
```

Comment:
Thanks for the doc update.  It might be better to land this in a separate branch, since this is not the first "and more" :)